pygen/pygen_src/riscv_instr_gen_config.py

```diff
@@ -413,9 +413,14 @@ def parse_args():
                        choices = [i.name for i in riscv_instr_group_t], nargs = '*')
     parse.add_argument('--directed_instr_0', help = 'directed_instr_0',
                        default = "riscv_int_numeric_corner_stream,4")
-    parse.add_argument('--stream_name_opts', help = 'stream_name_0',
+    parse.add_argument('--directed_instr_6', help = 'directed_instr_6',
+                       default = "riscv_jal_instr,4")
+    parse.add_argument('--stream_name_0', help = 'stream_name_0',
                        default = "riscv_load_store_rand_instr_stream")
-    parse.add_argument('--stream_freq_opts', help = 'stream_freq_0', default = 4)
+    parse.add_argument('--stream_name_6', help = 'stream_name_6',
+                       default = "riscv_int_numeric_corner_stream")
+    parse.add_argument('--stream_freq_0', help = 'stream_freq_0', default = 4)
+    parse.add_argument('--stream_freq_6', help = 'stream_freq_6', default = 4)
```

Comment:
I think we should dynamically parse these options (if they are passed in) rather than hardcoding each instruction stream to `--directed_instr_[0-9]`, e.g. if a user passes in `--directed_instr_0 riscv_jal_instr,5`, the config object should internally detect that a directed stream is being requested and parse this option and get the correct stream.

Reply:
Got it @udinator 
I will fix this ASAP.